h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -30,6 +32,11 @@
 
 @Ignore("Support for tests, but no actual tests here")
 public class TestUtil extends Iced {
+  {
```

Comment:
Can you please explain why do you need this? Our test framework already enables assertions on JVM level (-ea) flag.